### PR TITLE
Adds `Alias` Statement

### PR DESCRIPTION
**Pull request by markwpearce**
This is pretty cool!

`alias` provides a way to work around variable name shadowing... but as a side effect, it also give a way to shorthand just about any property/function you want.

```BrighterScript
alias get2 = get

namespace http
    'Do an HTTP request
    sub get()
        print get2.aa().data 'using `get2` aliased symbol here. it's now clear which item we intended to use
    end sub
end namespace

namespace get
    function aa()
        return {
            data: "abc"
        }
    end function
end namespace
```

transpiles to

```BrightScript
sub http_get()
    print get_aa().data
end sub

sub get_aa()
    return {
        data: "abc"
    }
end sub
```

it also allows this ... which is cool?:

```
alias lc = lcase
alias co = createObject

sub getModelTypeLower()
   lc(co("roDeviceInfo").getModelType())
end sub
```

it also allows this ... which is also cool?:

```
alias abgdz = alpha.beta.gamma.delta.zeta

sub foo()
   print abgdz.someConst
end sub
```

### CODE REVIEW

**Comment by markwpearce** [comment]
addresses #1113

File: src/files/BrsFile.spec.ts [hunk 1]
```diff
@@ -2877,6 +2877,215 @@ describe('BrsFile', () => {
...
+                    end function
+
+                    sub alpha_printDir()
+                        print "North"
+                        print "North"
+                    end sub
+                `);
+            });
+
+            it('can deep alias a namespaced thing', async () => {
```
Comment (TwitchBronBron) [inline src/files/BrsFile.spec.ts:3005]: Can you add a test for aliased class? And make sure we can `new` the class using the alias name? And that it transpiles properly?
i.e.:
```
alias Person2 = Person
sub test()
     dude = new Person2()
end sub
```

**Review by TwitchBronBron — CHANGES_REQUESTED** [review]
Looks great! Just a few small things

**Comment by markwpearce** [comment]
Updated to fixes review comments.

**Review by TwitchBronBron — APPROVED** [review]

**Pull request merged**